File: TechInterviews/Python/piglatin.py
def getPigLatin(word):
  firstLetter = word.slice(word, 1, 1)
  if firstLetter in ['a', 'e', 'i', 'o', 'y']
    word.append("ay")
  else
    word.append(firstLetter)
    word.append("ay")
  print word
  

File: TechInterviews/Javascript/piglatin.js
function getPigLatin(word) {
  var firstLetter = word.charAt(0);
  var pigLatin = "";
  if firstLetter in ['a', 'e', 'i', 'o', 'y']
    pigLatin = word.concat("ay");
  else
    pigLatin = word.splice(1).concat(firstLetter, "ay");
  console.log(pigLatin);
}
